src/components/forms/Textarea/Textarea.tsx

```diff
@@ -0,0 +1,158 @@
+import {FormField, Paragraph, Textarea as UtrechtTextarea} from '@utrecht/component-library-react';
+import type {TextareaProps as UtrechtTextareaProps} from '@utrecht/component-library-react/dist/Textarea';
+import clsx from 'clsx';
+import {useField, useFormikContext} from 'formik';
+import {useId, useLayoutEffect, useRef} from 'react';
+import {FormattedMessage} from 'react-intl';
+
+import HelpText from '@/components/forms/HelpText';
+import Label from '@/components/forms/Label';
+import Tooltip from '@/components/forms/Tooltip';
+import ValidationErrors from '@/components/forms/ValidationErrors';
+
+import './Textarea.scss';
+
+export interface TextareaProps {
+  /**
+   * The name of the form field/input, used to set/track the field value in the form state.
+   */
+  name: string;
+  /**
+   * The (accessible) label for the field - anything that can be rendered.
+   *
+   * You must always provide a label to ensure the field is accessible to users of
+   * assistive technologies.
+   */
+  label: React.ReactNode;
+  /**
+   * Required fields get additional markup/styling to indicate this validation requirement.
+   */
+  isRequired?: boolean;
+  /**
+   * Disabled fields get marked as such in an accessible manner.
+   */
+  isDisabled?: boolean;
+  /**
+   * Additional description displayed close to the field - use this to document any
+   * validation requirements that are crucial to successfully submit the form. More
+   * information that is contextual/background typically belongs in a tooltip.
+   */
+  description?: React.ReactNode;
+  /**
+   * Optional tooltip to provide additional information that is not crucial but may
+   * assist users in filling out the field correctly.
+   */
+  tooltip?: React.ReactNode;
+  /**
+   * Placeholder when no (default) value is available.
+   */
+  placeholder?: string;
+  /**
+   * Automatic expanding and shrinking of fields to fit their content.
+   */
+  autoExpand?: boolean;
+  /**
+   * Displays a countdown of the remaining amount of characters.
+   */
+  showCharCount?: boolean;
+  /**
+   * Needed for `showCharCount` to calculate the remaining amount of characters.
+   */
+  maxLength?: number;
+}
+
+const Textarea: React.FC<TextareaProps & UtrechtTextareaProps> = ({
+  name,
+  label = '',
+  isRequired = false,
+  description = '',
+  isDisabled = false,
+  autoExpand = false,
+  placeholder,
+  tooltip,
+  maxLength,
+  showCharCount = false,
+  ...extraProps
+}) => {
+  const {validateField} = useFormikContext();
+  const [{value, ...props}, {error = '', touched}] = useField<string | undefined>(name);
+  const id = useId();
+  const textareaRef = useRef<HTMLTextAreaElement | null>(null);
+
+  const invalid = touched && !!error;
+  const errorMessageId = invalid ? `${id}-error-message` : undefined;
+  const displayCharacterCount = showCharCount && maxLength !== undefined && value !== undefined;
```

Comment:
```suggestion
  const displayCharacterCount = showCharCount && value !== undefined;
```
you can enable `showCharCount` without setting a maximum length - in that case you just display the number of characters used. Only if both `showCharCount` and `maxLength` are provided do you show "x characters remaining".

Reply:
That makes sense... 👍